### PR TITLE
All LogRegrGD loss function divided by X.shape[0]

### PR DESCRIPTION
**Pull request by capac**
The Logistic Regression Gradient Descent loss function is now all divided by X.shape[0], just as presented in the Chapter 3 section _Converting an Adaline implementation into an algorithm for logistic regression_.

### CODE REVIEW

**Comment by rasbt** [comment]
Thanks for the PR!
